package.json

```diff
@@ -59,6 +59,8 @@
     "figlet": "^1.7.0",
     "fs-extra": "^11.2.0",
     "gradient-string": "^2.0.2",
+    "inquirer": "^9.2.22",
```

Comment:
question: why use this when we already have `enquirer`? I'm ok if we want to switch. We should just standardize on one though.